src/edit/DistortableCollection.Edit.js

```diff
@@ -303,28 +303,31 @@ L.DistortableCollection.Edit = L.Handler.extend({
         statusUrl = opts.statusUrl + data;
         // repeatedly fetch the status.json
         self.updateInterval = setInterval(function intervalUpdater() {
-          $.ajax(statusUrl + '?' + Date.now(), {// bust cache with timestamp
-            type: 'GET',
-            crossDomain: true,
-          }).done(function(data) {
-            opts.updater(data);
-          });
+          var request = new XMLHttpRequest();
+          request.onload = function(event) {
+            if (request.status === 200) {
+              opts.updater(JSON.parse(this.response));
+            }
+          };
+          request.open('GET', statusUrl + '?' + Date.now());
```

Comment:
We're going to want to copy over the `crossDomain: true` attributes from the jQuery as well! Especially important for the `POST` request.

Reply:
I thought the `crossDomain: true` option was only available on Ajax, I'm changing the resquests to the Request API anyways to keep them consistent with the rest. Just asking for further knowledge.

Reply:
yes you are right I was referring to the plain vanilla version of allowing cross domain requests. Great I will review that today :) thank you!!